sql/core/src/test/scala/org/apache/spark/sql/CTEInlineSuite.scala

```diff
@@ -703,6 +703,19 @@ abstract class CTEInlineSuiteBase
       checkErrorTableNotFound(e, "`tab_non_exists`", ExpectedContext("tab_non_exists", 83, 96))
     }
   }
+
+  test("SPARK-48307: not-inlined CTE references sibling") {
+    withTempView("t") {
```

Comment:
I can't find where you create or use this view

Reply:
ah I copied it from other test but didn't end up using it. Let me remove